src/components/Header/BaseHeaderMenuItem.test.jsx

```diff
@@ -14,17 +14,15 @@ import userEvent from "@testing-library/user-event";
 const user = userEvent.setup();
 
 describe("<BaseHeaderMenuItem/>", () => {
-  describe("Rendering and styling",  () => {
```

Comment:
Many of the changes in this file are due to Prettier. The only substantive change is adding the onClick property test.